### PR TITLE
A bit more on inclusive language.

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

**Review by ekline — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -827,7 +827,9 @@ consider](https://trac.ietf.org/trac/iesg/wiki/ExpertTopics).
 The IESG has made a
 [statement](https://www.ietf.org/about/groups/iesg/statements/on-inclusive-language/)
 recommending authors review [NISTIR 8366](https://doi.org/10.6028/NIST.IR.8366)
-for guidelines on using inclusive terminology.
+for guidelines on using inclusive terminology. Similar guidance exists for other
+RFC Streams. Authors are encouraged to familiarize themselves with and apply the
```
Comment (zaheduzzaman) [inline README.md:831]: reference to similar guidance will be highly appreciated here, I think.

Comment (larseggert) [reply README.md:831]: They all point to the same NIST doc

Comment (zaheduzzaman) [reply README.md:831]: Then I don't see the use of this sentence -"Similar guidance exists for other
RFC Streams". and kind of redundant.  and if they all are same then s/similar/same.

Comment (larseggert) [reply README.md:831]: Well, they all gave their own guidance, but it all points to NISTIR 8366

File: README.md [hunk 2]
```diff
@@ -827,7 +827,9 @@ consider](https://trac.ietf.org/trac/iesg/wiki/ExpertTopics).
 The IESG has made a
 [statement](https://www.ietf.org/about/groups/iesg/statements/on-inclusive-language/)
 recommending authors review [NISTIR 8366](https://doi.org/10.6028/NIST.IR.8366)
-for guidelines on using inclusive terminology.
+for guidelines on using inclusive terminology. Similar guidance exists for other
```
Comment (larseggert) [inline README.md:830]:
```suggestion
for guidelines on using inclusive terminology. Similar guidance pointing to [NISTIR 8366](https://doi.org/10.6028/NIST.IR.8366) exists for other
```

File: README.md (hunk 1 above)
Comment (zaheduzzaman) [reply README.md:831]: ok.

**Review by wkumari — APPROVED** [review]

**Comment by wkumari** [comment]
Thank you.

**Pull request merged**